kimchi/src/prover_index.rs

```diff
@@ -101,34 +96,30 @@ pub mod testing {
         gate::CircuitGate,
         lookup::{runtime_tables::RuntimeTableCfg, tables::LookupTable},
     };
-    use ark_poly::EvaluationDomain;
     use commitment_dlog::srs::endos;
-    use mina_curves::pasta::{pallas::Affine as Other, vesta::Affine, Fp};
+    use mina_curves::pasta::{pallas::Affine as Pallas, vesta::Affine as Vesta, Fp};
```

Comment:
makes me think that we should probably rename the structs called `Affine` in `mina-curves`

Reply:
I will open an issue.